sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DigitalTwinsImportJob.java

```diff
@@ -0,0 +1,224 @@
+ */
+@Fluent
+public final class DigitalTwinsImportJob {
+    /*
+     * The identifier of the bulk import job.
+     */
+    @JsonProperty(value = "id", access = JsonProperty.Access.WRITE_ONLY)
+    private String id;
+
+    /*
+     * The path to the input Azure storage blob that contains file(s)
+     * describing the operations to perform in the job.
+     */
+    @JsonProperty(value = "inputBlobUri", required = true)
+    private String inputBlobUri;
+
+    /*
+     * The path to the output Azure storage blob that will contain the errors
+     * and progress logs of import job.
+     */
+    @JsonProperty(value = "outputBlobUri", required = true)
+    private String outputBlobUri;
+
+    /*
+     * Status of the job.
+     */
+    @JsonProperty(value = "status", access = JsonProperty.Access.WRITE_ONLY)
+    private Status status;
+
+    /*
+     * Start time of the job. The timestamp is in RFC3339 format:
+     * `yyyy-MM-ddTHH:mm:ssZ`.
+     */
+    @JsonProperty(value = "createdDateTime", access = JsonProperty.Access.WRITE_ONLY)
+    private OffsetDateTime createdDateTime;
+
+    /*
+     * Last time service performed any action from the job. The timestamp is in
+     * RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.
+     */
+    @JsonProperty(value = "lastActionDateTime", access = JsonProperty.Access.WRITE_ONLY)
+    private OffsetDateTime lastActionDateTime;
+
+    /*
+     * End time of the job. The timestamp is in RFC3339 format:
+     * `yyyy-MM-ddTHH:mm:ssZ`.
+     */
+    @JsonProperty(value = "finishedDateTime", access = JsonProperty.Access.WRITE_ONLY)
+    private OffsetDateTime finishedDateTime;
+
+    /*
+     * Time at which job will be purged by the service from the system. The
+     * timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.
+     */
+    @JsonProperty(value = "purgeDateTime", access = JsonProperty.Access.WRITE_ONLY)
+    private OffsetDateTime purgeDateTime;
+
+    /*
+     * Details of the error(s) that occurred executing the bulk job.
+     */
+    @JsonProperty(value = "error")
+    private ErrorInformation errorInformation;
+
+    /**
+     * Creates an instance of BulkImportJob class.
+     *
+     * @param inputBlobUri the inputBlobUri value to set.
+     * @param outputBlobUri the outputBlobUri value to set.
+     */
+    @JsonCreator
+    public DigitalTwinsImportJob(
+            @JsonProperty(value = "inputBlobUri", required = true) String inputBlobUri,
+            @JsonProperty(value = "outputBlobUri", required = true) String outputBlobUri) {
+        this.inputBlobUri = inputBlobUri;
+        this.outputBlobUri = outputBlobUri;
+    }
+
+    /**
+     * Creates an instance of BulkImportJob class.
+     *
+     * @param id
+     * @param inputBlobUri
+     * @param outputBlobUri
+     * @param status
+     * @param createdDateTime
+     * @param lastActionDateTime
+     * @param finishedDateTime
+     * @param purgeDateTime
+     */
+    public DigitalTwinsImportJob(String id,
```

Comment:
Why does this constructor have to be public? E.g., the `finishedDateTime` is read-only and should never be specified by the customer. Can this be made package private?